frontend/src/pages/WorkflowitemDetails/WorkflowHistoryTab.js

```diff
@@ -24,7 +26,14 @@ function WorkflowitemHistoryTab({
       hasMore={hasMore}
       isLoading={isLoading}
       getUserDisplayname={getUserDisplayname}
-      fetchNext={() => fetchWorkflowitemHistory(projectId, subprojectId, workflowitemId, offset, limit)}
+      // fetchNext={() => fetchFirstWorkflowitemHistory(projectId, subprojectId, workflowitemId, offset, limit)}
+      fetchNext={() => {
+        if (isFirstHistoryCall) {
```

Comment:
If we only have one action creator (see above) we don't need to check anymore, wie simply pass the requested page (last page + 1)